.templates/php/default.mk

```diff
@@ -7,6 +7,15 @@ PHP_SOURCE_FILES = $(shell find . -name "*.php")
 
 ### COMMON stuff for all platforms
 
+BERP_VERSION = 1.3.0
+BERP_GRAMMAR = gherkin.berp
+
+define berp-generate-parser =
+-! dotnet tool list --tool-path /usr/bin | grep "berp\s*$(BERP_VERSION)" && dotnet tool update Berp --version $(BERP_VERSION) --tool-path /usr/bin
+berp -g $(BERP_GRAMMAR) -t $< -o $@ --noBOM
+endef
+
+
```

Comment:
Could that be part of the `gherkin/php/Makefile` rather than in all `php/default.mk`?

Reply:
It's in the default.mk for other languages too; I'm not clear why TBH